minui/minui.h

```diff
@@ -35,7 +35,7 @@ typedef struct {
 
 typedef GRSurface *gr_surface;
 
-int  gr_init(void);
+int  gr_init(bool blank);
```

Comment:
It is not immediately clear to me what is the purpose/meaning of the blank parameter. Could use a comment somewhere?